gapic-generator/templates/default/helpers/default_helper.rb

```diff
@@ -66,4 +66,17 @@ def format_number value
   def assert_locals *locals
     locals.each { |local| raise "missing local in template" if local.nil? }
   end
+
+  def start_line_spacer name = :default
+    (@line_spacers ||= {})[name] = false
+  end
+
+  def line_spacer name = :default
```

Comment:
this is way too opaque. Do we really need it as a helper? If so, needs documenting